### PR TITLE
Alarm Page Update

### PR DESCRIPTION
**Pull request by JingtingL**
Multiple Alarms, More User Modifiability

### CODE REVIEW

**Comment by JingtingL** [comment]
Don't merge

**Comment by JingtingL** [comment]
Still need to fix bug

File: client/src/Alarm.test.tsx
Comment (Shuzhengz) [inline client/src/Alarm.test.tsx:1]: put tests in `src/tests`

Comment (JingtingL) [reply client/src/Alarm.test.tsx:1]: Ok

**Review by Shuzhengz — CHANGES_REQUESTED** [review]
merge conflicts

**Review by JingtingL — COMMENTED** [review]
no test here

**Review by JingtingL — COMMENTED** [review]
no test here

**Comment by Shuzhengz** [comment]
the alarm on the homepage is broken, but i think we're changing the homepage layout so no need to fix this yet

image

**Comment by Shuzhengz** [comment]
I'm also not able to get the alarm to sound for some reason

**Review by Shuzhengz — APPROVED** [review]
doesn't seem like it is functional yet, but other than that the ux lgtm

o also prob add a remove alarm feature

**Comment by JingtingL** [comment]
> I'm also not able to get the alarm to sound for some reason

Have you pull???

**Comment by JingtingL** [comment]
Yup and there should be a remove alarm feature. Can you pull and send me what you see?

**Comment by Shuzhengz** [comment]
> Have you pull? The alarm sound should be there, under directly under the public folder??? Can you check for me and see if you see it?

O that was from last night, I can take a look at it again a bit later

**Comment by JingtingL** [comment]
Yeye I fixed it last night, not sure what happen sigh

**Review by Shuzhengz — APPROVED** [review]
Everything works now, lgtm!

File: client/src/components/pages/alarm.tsx [hunk 1]
```diff
@@ -1,100 +1,360 @@
...
+  sound: boolean;
+  active: boolean;
+  hasRung?: boolean; // Tracks if the alarm has already rung
+};
 
 const Alarm = () => {
-  
-  useEffect(() => {
-        document.title = "Sleep Buddies - Alarms";
-    }, []);
```
Comment (Shuzhengz) [inline client/src/components/pages/alarm.tsx:9]: @JingtingL i think this got deleted accidentally

**Pull request merged**